week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,14 +12,26 @@
  */
 
 class ATM {
-    // Add your code here
-
+  // Add your code here
+  constructor() {
+    this.balance = 100;
+  }
+  make_deposit(moneyIn) {
+    this.balance += moneyIn;
+  }
+  check_balance() {
+    console.log(this.balance);
+  }
+  make_withdraw(moneyOut) {
+    this.balance -= moneyOut;
+  }
 }
 
 let atm = new ATM(); // Create the ATM
 
 atm.make_deposit(200);
 atm.check_balance();
-atm.make_withdrawl(100);
+atm.make_withdraw(100);
 
-atm.make_withdrawl(500); // Your ATM should be able to handle this scenario
\ No newline at end of file
+atm.make_withdraw(500); // Your ATM should be able to handle this scenario
```

Comment:
Your ATM allows users to withdraw 500 even if they don't have enough money in their balance.

You should try and add some error handling so that if a user tries to withdraw more money then they have in their account then they should get an error message back.

Reply:
Oh. I will add some more handlings. Thank you!